ccm_web/client/src/models/models.ts

```diff
@@ -2,13 +2,31 @@
 Interfaces for common objects and entities (e.g. Globals, Course, Section, etc.)
 */
 
+export enum RoleEnum {
```

Comment:
You might add a comment above or below `RoleEnum` that says what each side is (left is client representation, right is Canvas representation?).

Reply:
Any consideration to making the left side not strings? I.e. `AccountAdmin`, so you can always do things like `RoleEnum.AccountAdmin`?
https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Reply:
The left side was intended to mirror how they are displayed in Canvas, I can mention that in a comment.

Reply:
Ah, didn't realize that. Comment would be helpful, thanks.